src/app.ts

```diff
@@ -168,6 +168,10 @@ export async function getApp(
     },
   })
 
+  await app.register(newrelicTransactionManagerPlugin, {
```

Comment:
Since DI config now relies on having NewRelic set by the plugin, it makes sense to instantiate it earlier than we run the DI initialization.